openmrs-client/src/main/java/org/openmrs/mobile/activities/settings/SettingsContract.java

```diff
@@ -36,6 +36,8 @@
         void setDarkMode();
 
         void chooseLanguage(String[] languageList);
+
+        void contactUs();
```

Comment:
```suggestion
        void setUpContactUsButton();
```
should be the name of an action